### PR TITLE
Finish metrics publisher to gcloud storage

### PR DESCRIPTION
**Pull request by hongchaodeng**

### CODE REVIEW

File: metrics/publisher/publisher.sh [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+KUBEMARK_LOG_GCLOUD_LOC=$1
+KUBEMARK_LOCAL_FILE="kubemark-log.txt"
+
+if ! command -v logplot >/dev/null 2>&1; then
+  echo "Please install logplot before running this script"
```
Comment (xiang90) [inline metrics/publisher/publisher.sh]: install logplot (github.com/coreos/kscale/logplot)

File: metrics/publisher/README.md [hunk 2]
```diff
@@ -1,11 +1,9 @@
 Metrics Publisher
 ======
 
-Metrics publisher takes all reports from a directory and uploads them to cloud storage.
+Metrics publisher takes a kubemark log, generates reports and uploads them to gcloud storage.
```
Comment (xiang90) [inline metrics/publisher/README.md]: a kubemark density test log (we might extend this in the future)

File: metrics/publisher/publisher.sh [hunk 3]
```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
```
Comment (xiang90) [inline metrics/publisher/publisher.sh]: rename this to publish (verb vs noun.)

**Comment by xiang90** [comment]
LGTM

**Comment by hongchaodeng** [comment]
Addressed all comments. Thanks!

**Pull request merged**